data/tr/4.0.affix

```diff
@@ -1,13 +1,11 @@
-")" "}" "]" ">" » 〉 ） 〕 》 】 ］ 』」 "’’" "’" ''.y '.y
-"%" "," "." 。 ":" ";" "?" "!" ‽ ؟ ？！ ….y ....y "”" "–" "‐" 、 ～
-¢ ₵ ™ ℠ : RPUNC+;
+")" "%" "," "." ":" ";" "?" "!" "''" "'" "'s": RPUNC+;
+"(" "$" "``": LPUNC+;
 
-"(" "{" "[" "<" « 〈 （ 〔 《 【 ［ 『 「 `` „ “ ‘ ''.x '.x ….x ....x
-¿ ¡ "$"
-£ ₤ € ¤ ₳ ฿ ₡ ₢ ₠ ₫ ৳ ƒ ₣ ₲ ₴ ₭ ₺  ℳ  ₥ ₦ ₧ ₱ ₰ ₹ ₨ ₪ ﷼ ₸ ₮ ₩ ¥ ៛ 호점
-† †† ‡ § ¶ © ® ℗ № "#": LPUNC+;
```

Comment:
Commenting on the diffs in this pull request: --  I'm going to merge this, but note above: the original LPUNC and RPUNC lists are more complete, and should probably be preserved.